javascript/es6/README.md

```diff
@@ -74,6 +74,18 @@ The game should also set a listener as soon as possible, so that it doesn't miss
     handlePageMessage(event);
   });
 })();
+
+// An example of handlePageMessage, which needs to handle any message from the page
+// See Messages to the Game below for more information
+function handlePageMessage(event) {
+  if (event.data.initGame === true) {
```

Comment:
@RobMSantos, this is just an example. The important part is to check for `event.data`, not just `event`.